.eslintrc.json

```diff
@@ -64,9 +70,14 @@
     "no-param-reassign": "off",
     "react/require-default-props": "off"
   },
-  "settings": {
-    "import/resolver": {
-      "typescript": {} // this loads <rootdir>/tsconfig.json to eslint
+  "overrides": [
+    {
+      "files": ["*.js"],
+      "rules": {
+        "@typescript-eslint/no-var-requires": "off",
```

Comment:
why disable these rules?

Reply:
It's applying to .js files where we are using `require(...)` rather than imports. This rule will only apply to .js files.